PS_REFRESH_VERSION

```diff
@@ -1 +1 @@
-9ad342b
+1b6f30c3623ec3e9690f5a5df16f073e0e9dc165
```

Comment:
I can't find this commit hash on PS nor PXC branch

Reply:
Changed.
1b6f30c3623ec3e9690f5a5df16f073e0e9dc165 is the hash of Percona-Server-5.6.48-88.0 tag
66735bc03f850e943a4bd92619be0914f10d625b is the hash of the commit
Both point to the same.

```
git show 1b6f30c3623ec3e9690f5a5df16f073e0e9dc165
tag Percona-Server-5.6.48-88.0
Tagger: Evgeniy Patlan <evgeniy.patlan@percona.com>
Date:   Wed May 13 16:36:25 2020 +0300

Percona Server release 5.6.48-88.0

commit 66735bc03f850e943a4bd92619be0914f10d625b (HEAD -> release-5.6.48-88.0, tag: Percona-Server-5.6.48-88.0, origin/release-5.6.48-88.0, origin/release-5.6.48-87.1)
Merge: 8e903b24f04 41d5569727a
Author: Przemyslaw Skibinski <przemyslaw.skibinski@percona.com>
Date:   Thu May 7 11:30:00 2020 +0200

    Merge pull request #3805 from inikep/PS-7020-5.6
    
    PS-7020: convert MTR tests from python2 to python3 (5.6)

```
Let's stick to the 2nd hash to avoid confustion.